docs/backups-compress.md

```diff
@@ -1,17 +1,10 @@
 # Enable compression for backups
 
-There is a possibility to enable [LZ4 compression :octicons-link-external-16:](https://en.wikipedia.org/wiki/LZ4_(compression_algorithm))
-for backups.
+You can enable [LZ4 compression :octicons-link-external-16:](https://en.wikipedia.org/wiki/LZ4_(compression_algorithm))
+for backups if you run Percona XtraDB Cluster 8.0.
```

Comment:
and PXC 8.4. As we will be adding other PXC versions in future and new versioning is used, I would stick to This feature is available only with Percona XtraDB Cluster 8.0 and higher